projboard/templates/base.html

```diff
@@ -0,0 +1,66 @@
+{% load static %}
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+	  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+    
+    <!-- Smarticle CSS -->
+	  <link rel="stylesheet" type="text/css" href="{% static 'css/base.css' %}">
+	
+    <title>Smarticle</title>
+   
+  </head>
+  <body>
+  <!--Top menu-->
+  <div class="container-fluids">
+    <nav class=" navbar navbar-expand-md sticky-top navbar-dark bg-dark ">
+      <div class="container">
+        <a class="navbar-brand"><span class="text-primary"><img src="{% static 'images/Smarticle_logo_prev_ui.png' %}" width="55" height="55" alt="logo"></span></a>
+	      <a class="navbar-brand" href="/">Smarticle</a>
+
+        <button
+          class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+	
+        <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Search Article</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+            </ul>
+        </div>
+    </nav>
+
+
+    <main>
+      <br/><br/>
+      <div class="container-fluid mt-3 ps-0 pe-0 h-100 d-inline-block">
+        {% block content %}{% endblock %}
+      </div>
+    </main>
+   
+    <!--footer-->
+    <footer class="bg-dark text-center text-lg-start mt-3">
+          <div class="text-center p-3">
+              <a class="text-light">© 2022: Smarticle </a>
```

Comment:
Consider to change the a tag to p tag

Reply:
I prefer to leave it as tag "a" because it adds a line below

Reply:
Ok, np.